testing/GenFiles/TestGen.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+#define FILEBUFFS 20
+#define BYTESELECTWIDTH 6
+#define INDEXWIDTH	14
+#define TAGWIDTH	12
+
+int main(int argc, char *argv[]) {
+	if(argc < 3) printf("TestGen <input file> <output file>\n");
```

Comment:
This should exit if we don't give it 3 argc, right now it doesn't.

Reply:
Woops. I meant for it to but then just forgot to type it